kore/src/Kore/Log/DebugSolver.hs

```diff
@@ -73,9 +73,11 @@ instance Pretty DebugSolverRecv where
 
 instance Entry DebugSolverSend where
     entrySeverity _ = Debug
+    helpDoc _ = "log SMT solver queries"
```

Comment:
`DebugSolverSend` does not log only queries, but every command sent to the solver:

```suggestion
    helpDoc _ = "log commands sent to SMT solver"
```